heathskin/entity.py

```diff
@@ -98,4 +94,4 @@ def __init__(self, tag_name, old_value, new_value):
         self.new_value = new_value
 
     def __repr__(self):
-        return "EntityChangeEvent tag_name=" + str(self.tag_name) + " old:" + str(self.old_value) + " new:" + str(self.new_value)
\ No newline at end of file
+        return "EntityChangeEvent tag_name={0} old={1} new={2}".format(str(self.tag_name), str(self.old_value), str(self.new_value))
```

Comment:
they'll get formatted to str() automatically by format().

also, you can commit the 0, 1, 2 inside the {}'s if you're putting everything in order anyway

also, 80cols